.github/actions/checkov-scans-v1/action.yml

```diff
@@ -0,0 +1,35 @@
+name: checkov-scans
+description: Checkov scans cloud infrastructure configurations to find misconfigurations before they're deployed and sends a result to Prisma Cloud.
+
+inputs:
+  output-file-path:
+    description: Output file path of Checkov scans
+    required: true
+  prisma-access-key:
+    description: Prisma Cloud access key
+    required: true
+  prisma-secret-key:
+    description: Prisma Cloud secret key
+    required: true
+  prisma-api-url:
+    description: Prisma Cloud API URL
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so follow-up steps can access it
+    - uses: actions/checkout@v4
+    # The checkov scan
+    - name: Run Prisma Cloud
+      id: prisma-cloud
+      uses: bridgecrewio/checkov-action@v12
```

Comment:
We'll need to make sure that we pin this version to a specific commit SHA

Reply:
but we don't need to create any PR or do something with GitHub

Reply:
It's a [policy for security](https://github.com/dequelabs/axe-api-team/blob/main/policies/github-actions.md#pin-3rd-party-actions-to-commit-sha) to pin 3rd party actions to a commit SHA. Git tags are changeable so `v12` could be updated to include malicious code in the future.

Reply:
Yeah good catch - most 3rd party actions should be pinned to a particular hash and have a dependabot identifier so they can get automatically updated like so: 

```
uses: bridgecrewio/checkov-action@99bb2caf247dfd9f03cf984373bc6043d4e32ebf # tag=v12
```

The SHA would would be whatever is latest at the time of creating this PR in this case its 99bb.. on the v12 tag. the `# tag=v12` tells dependabot to update to the latest SHA on the v12 tag if any

Reply:
@michael-siek 
set the SHA for the checkov-action